jobs/haproxy/spec

```diff
@@ -704,6 +704,9 @@ properties:
   ha_proxy.backend_config:
     description: |
       Raw HAProxy config that will be added to the default HTTP + routed HTTP backend definitions, provided either as a multiline text blob or as an array of lines.
+  ha_proxy.backend_config_targeted:
+      description: |
```

Comment:
Please update the description of the new property

Reply:
I think it still not precise enough. It seems to be a map of backend names to raw config properties. It can be very helpful to provide an example in the spec.